### PR TITLE
New Polish Cheat Sheet

### PR DESCRIPTION
**Pull request by Tykzz**
IA page: https://duck.co/ia/view/polish_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@Tykzz Thanks for contributing and welcome to DuckDuckHack!

Seeing as your PR is for a new IA, I need the PR to only contain the new files you're adding. Let's keep the initial Polish Cheat Sheet to the standard implementation for now.

Then, please make a separate PR adding the audio functionality. I'd like to discuss it further with you and my team. I suspect we'l need to use our built-in SoundManager lib to handle the audio so we're sure it works across all browsers

**Comment by Tykzz** [comment]
Right, I should've separated these changes. Sorry, I'm still learning how to use git.
Should I add two new PRs or change this one and add one new?

**Comment by gautamkrishnar** [comment]
@Tykzz you don't need to close this pr, please create a new pr for play pronunciation feature
just type the following commands in codio:
```bash
cd zeroclickinfo-goodies
git checkout master
git rm share/goodie/cheat_sheets/language.handlebars
git rm  share/goodie/cheat_sheets/cheat_sheets.js
git rm  share/goodie/cheat_sheets/language.handlebars
git add --all .
git commit -m "removed files from git"
```

For creating a new PR for audio playback feature use:
```bash
git checkout -b audio_feature
git add share/goodie/cheat_sheets/language.handlebars
git add  share/goodie/cheat_sheets/cheat_sheets.js
git add  share/goodie/cheat_sheets/language.handlebars
git rm  share/goodie/cheat_sheets/json/language/polish.json
git add --all .
git commit -m "added audio_feature"
git push origin audio_feature
```

**Comment by gautamkrishnar** [comment]
@Tykzz one tip for you:
https://github.com/Tykzz/zeroclickinfo-goodies/pull/1
You actually don't actually need to create a new pull request in your repo for making your fork uptodate with the main repository. You can use this command instead:
`git pull https://github.com/duckduckgo/zeroclickinfo-goodies.git master`

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Polish cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/polish_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Polish cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/polish_cheat_sheet_mobile.png)

**Comment by Tykzz** [comment]
@gautamkrishnar I worked on `polish-cheatsheet` branch so I solved it kinda differently. Also I think removing the files in this PR wouldn't be good so I managed to leave just one change - adding the JSON file.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Polish cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/polish_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@Tykzz it looks like it's successfully deployed to Beta: https://beta.duckduckgo.com/?q=Polish%20cheat%20sheet

Are you a native Polish speaker?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Polish cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/polish_cheat_sheet_mobile.png)

**Comment by Tykzz** [comment]
@moollaza I am. Can't wait to see DuckDuckGo fully internationalized!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Polish cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/polish_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/language/polish.json [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        }],
+        "General Conversation": [{
+            "key": "Yes",
+            "val": "Tak"
+        }, {
+            "key": "No",
+            "val": "Nie"
+        }, {
+            "key": "Please / You’re welcome",
+            "val": "Proszę"
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/language/polish.json]: According to a Polish gentleman I work with, this should be: `Proszę/Proszę bardzo` for `You're Welcome`

Comment (Tykzz) [inline share/goodie/cheat_sheets/json/language/polish.json]: @mintsoft `Proszę bardzo` would be `You're very welcome` and it's actually the next phrase in the [source](http://mowicpopolsku.com/polish-vocabulary-phrases/polish-conversations) which I decided not to include.

**Comment by Tykzz** [comment]
@moollaza Shouldn't be labeled `New Instant Answer` instead of `Improvement`? I guess it has remained because the audio functionality that was initially incorporated in this PR.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Polish cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/polish_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@Tykzz apologies for the delay! This LGTM :+1:

**Pull request merged**